server/aws-lsp-partiql/tsconfig.json

```diff
@@ -3,7 +3,8 @@
     "compilerOptions": {
         "rootDir": "./src",
         "outDir": "./out",
-        "allowJs": true
+        "allowJs": true,
+        "esModuleInterop": true
```

Comment:
What do we need this for?

Reply:
The module '"web-tree-sitter"' can only be default-imported using the 'esModuleInterop' flag. So added this setting line here.